scripts/create-test-segments.js

```diff
@@ -58,14 +58,25 @@ const createTestSegments = {
 
     const segmentDataExportStrings = Object.keys(segmentData).reduce((acc, key) => {
       // use a function since the segment may be cleared out on usage
-      acc.push(`export const ${key} = () => base64ToUint8Array('${segmentData[key]}');`);
+      acc.push(`export const ${key} = () => {
+        cache.${key} = cache.${key} || base64ToUint8Array('${segmentData[key]}');
```

Comment:
We were converting the raw data to from base64 every time, which caused ie 11 and edge to fail in various tests. Same with utf16 to string below. Now we cache after we do it once.